cmd/lncli/cmd_payments.go

```diff
@@ -1556,7 +1555,15 @@ var buildRouteCommand = cli.Command{
 	Name:     "buildroute",
 	Category: "Payments",
 	Usage:    "Build a route from a list of hop pubkeys.",
-	Action:   actionDecorator(buildRoute),
+	Description: `
+	Builds a sphinx route for the supplied hops (public keys). Make sure to
+	use a custom final_cltv_delta to create the route depending on the
+	restrictions in the invoice otherwise LND will use its default specified
+	via the bitcoin.timelockdelta setting (default 80).
```

Comment:
Should we specifically mention the `INCORRECT_OR_UNKNOWN_PAYMENT_DETAILS` error here perhaps?